### PR TITLE
Add tutorials for additional Minio Gateway modes

### PR DESCRIPTION
**Pull request by michal-hudy**
**Description**

Changes proposed in this pull request:

- Added Azure Gateway mode
- Added AWS S3 mode
- Added Alibaba OSS Gateway mode

**Related issue(s)**

#3973

### CODE REVIEW

File: docs/asset-store/08-03-gateway-s3-mode-set-up.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Amazon Web Services (AWS)](https://aws.amazon.com) account
+
+## Steps
+
+You can set Minio to the AWS S3 Gateway mode both during and after Kyma installation. In both cases, you need to create and configure a access key for AWS account, apply a Secret and Config Map with an override onto a cluster or Minikube, and trigger the Kyma installation process. 
+
+>**CAUTION:** Buckets created in Minio without using Bucket CRs are not recreated or migrated while switching to the Minio Gateway mode.
+
+>**CAUTION:** AWS S3 Gateway Mode is testes only manually.
```
Comment (derberg) [inline docs/asset-store/08-03-gateway-s3-mode-set-up.md]: Please move those cautions to top of each doc
and write openly that this instruction was tested manually against kyma 1.6 while others are regularly tested on pipelines. 


maybe we should link here to future ticket?

Comment (michal-hudy) [reply docs/asset-store/08-03-gateway-s3-mode-set-up.md]: @derberg take a look if it is OK now. Yes I can link, but first I need to create it 😃

File: docs/asset-store/08-03-gateway-s3-mode-set-up.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+>**CAUTION:** Buckets created in Minio without using Bucket CRs are not recreated or migrated while switching to the Minio Gateway mode.
+
+### Create a AWS access key
+
+Create a AWS access key for IAM user. Follow these steps:
+
+1. Access the [AWS Identity and Access Management console](https://console.aws.amazon.com/iam/)
+2. In the left navigation panel, choose **Users**.
+3. Choose the user whose access keys you want to create, and then choose the **Security credentials** tab.
```
Comment (kazydek) [inline docs/asset-store/08-03-gateway-s3-mode-set-up.md]:
```suggestion
3. Choose the user whose access keys you want to create, and select the **Security credentials** tab.
```

File: docs/asset-store/08-03-gateway-s3-mode-set-up.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+4. In the **Access keys** section, choose **Create access key**.
```
Comment (kazydek) [inline docs/asset-store/08-03-gateway-s3-mode-set-up.md]:
```suggestion
4. In the **Access keys** section, select **Create access key**.
```

File: docs/asset-store/08-03-gateway-s3-mode-set-up.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+5. Export the access key and secret key as an environment variable:
```
Comment (kazydek) [inline docs/asset-store/08-03-gateway-s3-mode-set-up.md]:
```suggestion
5. Export the access and secret keys as environment variables:
```

File: docs/asset-store/08-03-gateway-s3-mode-set-up.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+    ```bash
+    export AWS_ACCESS_KEY={your-access-ID}
+    export AWS_SECRET_KEY={your-secret-key}
+    ```
+
+### Configure Minio Gateway mode
+
+Export AWS S3 service endpoint (https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region) as an environment variable:
```
Comment (kazydek) [inline docs/asset-store/08-03-gateway-s3-mode-set-up.md]:
```suggestion
Export an AWS S3 service endpoint (https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region) as an environment variable:
```

File: docs/asset-store/08-03-gateway-s3-mode-set-up.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
+---
+title: Set Minio to the AWS S3 Gateway mode
+type: Tutorials
+---
+
+>**CAUTION:** AWS S3 Gateway Mode was tested only manually on Kyma 1.6. Currently, we don't have an automated pipeline like for other gateway modes.
```
Comment (kazydek) [inline docs/asset-store/08-03-gateway-s3-mode-set-up.md]:
```suggestion
>**CAUTION:** AWS S3 Gateway mode was only tested manually on Kyma 1.6. Currently, there is no automated pipeline for it in Kyma.
```

File: docs/asset-store/08-04-gateway-oss-mode-set.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
+---
+title: Set Minio to the Alibaba OSS Gateway mode
+type: Tutorials
+---
+
+>**CAUTION:** Alibaba OSS Gateway Mode was tested only manually on Kyma 1.6. Currently, we don't have an automated pipeline like for other gateway modes.
```
Comment (mmitoraj) [inline docs/asset-store/08-04-gateway-oss-mode-set.md]:
```suggestion
>**CAUTION:** Alibaba OSS Gateway Mode was only tested manually on Kyma 1.6. Currently, there is no automated pipeline for it in Kyma
```

File: docs/asset-store/08-04-gateway-oss-mode-set.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+You can set Minio to the Alibaba OSS Gateway mode both during and after Kyma installation. In both cases, you need to create and configure a access key for Alibaba Cloud account, apply a Secret and Config Map with an override onto a cluster or Minikube, and trigger the Kyma installation process. 
+
+>**CAUTION:** Buckets created in Minio without using Bucket CRs are not recreated or migrated while switching to the Minio Gateway mode.
+
+### Create a Alibaba Cloud access key
+
+Create a Alibaba Cloud access key for user. Follow these steps:
+
+1. Access the [Resource Access Management console](https://ram.console.aliyun.com)
+2. In the left navigation panel, choose **User**.
```
Comment (mmitoraj) [inline docs/asset-store/08-04-gateway-oss-mode-set.md]:
```suggestion
2. In the left navigation panel, select **User**.
```

File: docs/asset-store/08-04-gateway-oss-mode-set.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+5. Export the access key and secret key as an environment variable:
+
+    ```bash
+    export ALIBABA_ACCESS_KEY={your-access-ID}
+    export ALIBABA_SECRET_KEY={your-secret-key}
+    ```
+
+### Configure Minio Gateway mode
+
+Export Alibaba OSS service endpoint (https://www.alibabacloud.com/help/doc-detail/31837.htm) as an environment variable:
```
Comment (mmitoraj) [inline docs/asset-store/08-04-gateway-oss-mode-set.md]:
```suggestion
Export an Alibaba OSS service [endpoint](https://www.alibabacloud.com/help/doc-detail/31837.htm) as an environment variable:
```

File: docs/asset-store/08-04-gateway-oss-mode-set.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+
+### Create a Alibaba Cloud access key
+
+Create a Alibaba Cloud access key for user. Follow these steps:
+
+1. Access the [Resource Access Management console](https://ram.console.aliyun.com)
+2. In the left navigation panel, choose **User**.
+3. Choose the user whose access keys you want to create.
+4. Click **Create AccessKey** in the **User AccessKey** section.
+5. Export the access key and secret key as an environment variable:
```
Comment (mmitoraj) [inline docs/asset-store/08-04-gateway-oss-mode-set.md]:
```suggestion
5. Export the access and secret keys as environment variables:
```

File: docs/asset-store/08-02-gateway-azure-mode-set-up.md [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure)
+- [Microsoft Azure](http://azure.com) subscription
+
+## Steps
+
+You can set Minio to the Azure Blob Storage Gateway mode both during and after Kyma installation. In both cases, you need to create and configure an Azure storage account, apply a Secret and ConfigMap with an override onto a cluster or Minikube, and trigger the Kyma installation process. 
+
+>**CAUTION:** Buckets created in Minio without using Bucket CRs are not recreated or migrated while switching to the Minio Gateway mode.
+
+### Setup Azure Blob Storage resources
```
Comment (kazydek) [inline docs/asset-store/08-02-gateway-azure-mode-set-up.md]:
```suggestion
### Set up Azure Blob Storage resources
```

File: docs/asset-store/08-03-gateway-s3-mode-set-up.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+You can set Minio to the AWS S3 Gateway mode both during and after Kyma installation. In both cases, you need to create and configure an access key for the AWS account, apply a Secret and ConfigMap with an override onto a cluster or Minikube, and trigger the Kyma installation process. 
+
+>**CAUTION:** Buckets created in Minio without using Bucket CRs are not recreated or migrated while switching to the Minio Gateway mode.
+
+### Create an AWS access key
+
+Create an AWS access key for an IAM user. Follow these steps:
+
+1. Access the [AWS Identity and Access Management console](https://console.aws.amazon.com/iam/)
+2. In the left navigation panel, choose **Users**.
```
Comment (kazydek) [inline docs/asset-store/08-03-gateway-s3-mode-set-up.md]:
```suggestion
2. In the left navigation panel, select **Users**.
```

File: docs/asset-store/08-04-gateway-oss-mode-set.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
+---
+title: Set Minio to the Alibaba OSS Gateway mode
+type: Tutorials
+---
+
+>**CAUTION:** Alibaba OSS Gateway Mode was only tested manually on Kyma 1.6. Currently, there is no automated pipeline for it in Kyma
```
Comment (mmitoraj) [inline docs/asset-store/08-04-gateway-oss-mode-set.md]:
```suggestion
>**CAUTION:** Alibaba OSS Gateway Mode was only tested manually on Kyma 1.6. Currently, there is no automated pipeline for it in Kyma.
```

File: docs/asset-store/08-04-gateway-oss-mode-set.md [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+
+You can set Minio to the Alibaba OSS Gateway mode both during and after Kyma installation. In both cases, you need to create and configure an access key for your Alibaba Cloud account, apply a Secret and a ConfigMap with an override onto a cluster or Minikube, and trigger the Kyma installation process. 
+
+>**CAUTION:** Buckets created in Minio without using Bucket CRs are not recreated or migrated while switching to the Minio Gateway mode.
+
+### Create an Alibaba Cloud access key
+
+Create an Alibaba Cloud access key for a user. Follow these steps:
+
+1. Access the [Resource Access Management console](https://ram.console.aliyun.com)
```
Comment (mmitoraj) [inline docs/asset-store/08-04-gateway-oss-mode-set.md]:
```suggestion
1. Access the [Resource Access Management console](https://ram.console.aliyun.com).
```

**Pull request merged**